### PR TITLE
Make changes to Java data flow

### PR DESCRIPTION
**Pull request by prateekma**

### CODE REVIEW

File: src/main/java/org/photonvision/common/SimplePipelineResult.java [hunk 1]
```diff
@@ -18,32 +18,79 @@
...
+   * Returns the latency in the pipeline.
+   *
+   * @return The latency in the pipeline.
+   */
   public double getLatencyMillis() {
     return latencyMillis;
   }
 
+  /**
+   * Returns whether the pipeline has targets.
```
Comment (Bankst) [inline src/main/java/org/photonvision/common/SimplePipelineResult.java:75]: Whether or not?

File: src/test/java/org/photonvision/common/PacketTest.java [hunk 2]
```diff
@@ -40,49 +40,44 @@
...
-class BytePackableTest {
+class PacketTest {
   @Test
   void testSimpleTrackedTarget() {
     var target = new SimpleTrackedTarget(3.0, 4.0, 9.0, -5.0,
         new Pose2d(1, 2, new Rotation2d(1.5)));
-    byte[] packed = target.toByteArray();
-
-    System.out.println(Arrays.toString(packed));
+    var p = new Packet(SimpleTrackedTarget.PACK_SIZE_BYTES);
```
Comment (Bankst) [inline src/test/java/org/photonvision/common/PacketTest.java:50]: better variable naming here?

**Review by Bankst — APPROVED** [review]

File: src/main/java/org/photonvision/common/SimplePipelineResult.java (hunk 1 above)
Comment (prateekma) [reply src/main/java/org/photonvision/common/SimplePipelineResult.java:75]: will resolve in restructuring PR

File: src/test/java/org/photonvision/common/PacketTest.java (hunk 2 above)
Comment (prateekma) [reply src/test/java/org/photonvision/common/PacketTest.java:50]: will resolve in restructuring PR

**Pull request merged**